morceus-data/latin/stems/nominals/ls.nom

```diff
@@ -172212,7 +172321,10 @@ quattu^or-vi^ri		<orth type=alt>quat-</orth>
 
 
 quaxa_re		<itype>ranae dicuntur, cum vocem mittunt</itype>
-que^	
+que^
```

Comment:
is this the enclitic? Can we add an `enclitic` label here so it's clear?

Reply:
added enclitic label